docs/language.md

```diff
@@ -81,6 +81,24 @@ This has dramatic implications for how simple and performant we can make our par
 
 _N.B.:_ The defining aspect of blocks are their semantics and the isolation mechanism they provide; in other words, their identity. On the other hand, where their data is stored is a more liberal aspect. Blocks support more than just static local data (via JSON literals inside the HTML comment or within the block's HTML), and more mechanisms (_e.g._, global blocks or otherwise resorting to storage in complementary `WP_Post` objects) are expected. See [attributes](../reference/attributes) for details.
 
+## The Anatomy of a Serialized Block
+
+When blocks are saved to the content after the editing session, the different attributes—depending on the nature of the block—are serialized to these explicit comment delimiters.
```

Comment:
The word "different" feels out of place here. Should we drop it, or be more specific about those attributes which are serialized in the delimiters?